src/v/kafka/protocol/fetch.h

```diff
@@ -488,10 +488,16 @@ struct op_context {
 
 struct fetch_config {
```

Comment:
please describe in the commit message high level descriptions of what is changing to fetch handling.